src/Misc/layoutbin/macOSRunInvoker.js

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+// Copyright (c) GitHub. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+const { spawn } = require('child_process');
+var shell = process.argv[2];
+var args = process.argv.slice(3);
+var launch = spawn(shell, args, { stdio: 'inherit' });
+launch.on('exit', function (code) {
```

Comment:
make sure you got the right one (exit vs closed). I think closed is when streams have been released. So i think exit is correct in this case.